### DISCUSSION TITLE
【求助】为什么每次一更新助手就刷不出来了，但是我还能发帖

### DISCUSSION THREAD

**Discussion by NOVAQUQ** [share121/inter-knot#2389] [其他]
![Screenshot_2024-08-16-14-16-20-77_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/227cd565-f54e-40e1-8bba-f5e76186b34b)
为啥啊？咋办啊？救命啊！

**Comment by liuying5201314** [comment]
这玩意有点吃网速，网有一点不行就看不了
![Uploading 9725dcf6-98f2-41ee-a309-b510b8a67326.png…]()

**Reply by Dave-12138** [reply]
我写的表情包脚本设计上不是让你把分类图标拖到输入框里啊。。。
这样用会有上传的时间，而且别人看的时候加载也慢，还没缓存，别人五分钟就要重新加载一次

**Comment by liuying5201314** [comment]
我这也好卡，表情包都发不出来
![Uploading 69f38b1a-3de8-4d5a-8f13-5b1b95511035.png…]()

**Reply by NOVAQUQ** [reply]
![Screenshot_2023-11-22-00-18-42-71_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/f795e86d-b907-49c0-aa99-a7d6e77f091d)
难受😣

**Reply by liuying5201314** [reply]
难受加一
![364d144b-18e4-4b2b-ad4b-53f31dc31fbb](https://github.com/user-attachments/assets/900474c5-ec6d-487e-8747-c38a0e1be992)
![1e3bdeaa-2a43-4b49-ad5e-576a1e27b640](https://github.com/user-attachments/assets/f0296f24-a34c-4f8b-a998-b414e44d8d7d)

**Comment by 6078HITHEDAY** [comment]
ＧｉｔＨｕｂ的特性，嗯，讲不明白，服务器在　_github_　，就这么说吧

**Reply by NOVAQUQ** [reply]
那岂不是丸辣😱能不能上线全看GH的心情

**Comment by dthftjf** [comment]
已经卡一天的我陷入沉思

**Comment by liuying5201314** [comment]
我倒是没这么卡，不过为啥你的回复发到了我的邮箱？






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "share121/inter-knot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月16日(星期五) 下午4:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [share121/inter-knot] 【求助】为什么每次一更新助手就刷不出来了，但是我还能发帖 (Discussion #2389)





 
已经卡一天的我陷入沉思

**Reply by Dave-12138** [reply]
你subscribe了这个讨论串，所有新回复都会发个邮件通知你
